<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QMainWindow" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WelcomeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome in Transportation MarketPlace</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Registration</string>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>301</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="Cargo_Reg">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose the role for Registration:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Register as Cargo Owner</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose another
 role</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>231</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Press proceed to Continue</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Company_Reg">
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose another
 role</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>261</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Register as Transportation Company</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose the role for Registration:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>231</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Press proceed to Continue</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Driver_Reg">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose the role for Registration:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>261</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Register as a Driver</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose another
 role</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>231</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Press proceed to Continue</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="LoginStack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Login for Cargo Owner</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Email Address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Owner_Email">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>64</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Owner_Password">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="Login_Owner">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ChooseAnotherRole1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose another
 role</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Login for Transportation Company</string>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Email Address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Company_Email">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>64</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Company_Password">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="Login_Company">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ChooseRole2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose another
 role</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Login for Driver</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Email Address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Driver_Email">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>64</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Driver_Password">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="Login_Driver">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ChooseRole3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose another
 role</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the role for login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
